tests/functional/conftest.py

```diff
@@ -2,6 +2,10 @@
 
 from flask import url_for
 
+from tests.conftest import responses, config  # noqa: F401
```

Comment:
This import is not necessary - change `config` in `tests/conftest.py` to a fixture instead. The `responses` fixture is already imported by pytest